pkg/flowaggregator/flowaggregator.go

```diff
@@ -587,6 +590,10 @@ func (fa *flowAggregator) flowExportLoopProxy(stopCh <-chan struct{}) {
 				break
 			}
 			proxyRecords(msg)
+		case <-flushTicker.C:
+			if err := fa.flushExporters(); err != nil {
```

Comment:
According to the change in this PR: https://github.com/antrea-io/antrea/pull/7006, does it mean there will be log flood if no collector address is provided.

Reply:
No, it should not generate any logs. `flushExporters` will be a no-op if there is no exporter:
```go
func (fa *flowAggregator) flushExporters() error {
        if fa.ipfixExporter != nil { // ADDED: this will be nil when there is no exporter, and we will ultimately return nil (no error)
                if err := fa.ipfixExporter.Flush(); err != nil {
                        return err
                }
        }
        // Other exporters don't leverage Flush for now, so we skip them.
        return nil
}
```